server/world/item.go

```diff
@@ -26,6 +26,9 @@ type CustomItem interface {
 	Texture() image.Image
 	// Category is the category the item will be listed under in the creative inventory.
 	Category() category.Category
+	// ImageSize is the width/height size value of the image which will be used to correct item render scale. We assume width and height are same.
+	// Minecraft uses 16x16 for their image. If your image uses 16x16 then you could return 16.
+	ImageSize() uint8
```

Comment:
This isn't the most reliable method, you can calculate the size of the texture using the Bounds() method on the image.Image interface